src/components/forms/CreateSmsTemplate/form-actions.ts

```diff
@@ -0,0 +1,7 @@
+export const calculateHowManySmsMessages = (val: number) => {
```

Comment:
Please can we change `val` to `characterCount` and add a comment linking to:

https://www.notifications.service.gov.uk/pricing/text-messages#long-text-messages

As although this is correct, it's not clear **why** this is that breakdown.

Reply:
Renamed `val` to `characterCount`. I also added a link to the above on the func description.